src/fish/devcontainer-feature.json

```
@@ -9,6 +9,11 @@
       "type": "boolean",
       "default": true,
       "description": "Install Fisher plugin manager"
+    },
+    "fisher_plugins": {
```

It should be camelCase.

```suggestion
    "fisherPlugins": {
```